### ISSUE TITLE
FreeFileSync is not accessable for visually impaired users who use it by screen-reader

### ISSUE THREAD

**Issue by shadyar** [flathub/org.freefilesync.FreeFileSync#104]
Visually impaired users use Screen-reader to study and control their computer and mobiles.
While installing Next Back and Cancel buttons also all radio buttons are not properly read by a screen-reader. It seems a character set is used that is not UTF-8.
After installing, when the windows opens, for example Buttons labels are not readable for screen-readers. 
Please correct this issue in the program.
To test FreeFileSync, you can use NVDA. This is a free and open source screen-reader.
http://www.NVAccess.org
For further information about Accessibility please take a look at WCAG v2.0 
https://www.w3.org/WAI/standards-guidelines/wcag/

**Comment by kparal** [comment]
Hi, unfortunately, this is not the place to file issues against FreeFileSync itself. This is just a project concerned with packaging it for Flathub. To contain the actual FFS developer, you need to go to https://freefilesync.org/forum/ .

**Comment by shadyar** [comment]
Ok, I'll do it. Thanks.
